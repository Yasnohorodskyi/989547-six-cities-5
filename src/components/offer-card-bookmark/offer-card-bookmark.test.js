import React from "react";
import renderer from "react-test-renderer";

import BookmarkCard from "./offer-card-bookmark";

jest.mock(`../offer-card/offer-card`, () => `OfferCard`);

it(`Should bookmark offer card render`, () => {
  const tree = renderer
  .create(<BookmarkCard />).toJSON();

  expect(tree).toMatchSnapshot();
});
