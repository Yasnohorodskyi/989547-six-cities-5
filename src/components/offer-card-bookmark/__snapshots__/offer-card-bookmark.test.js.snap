// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should bookmark offer card render 1`] = `
<OfferCard
  block="favorites"
  className="favorites__card"
  imgHeight="110"
  imgWidth="150"
/>
`;
